components/infobox/wikis/deadlock/infobox_item_custom.lua

```diff
@@ -0,0 +1,220 @@
+---
+-- @Liquipedia
+-- wiki=deadlock
+-- page=Module:Infobox/Item/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Logic = require('Module:Logic')
+local Namespace = require('Module:Namespace')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector')
+local Item = Lua.import('Module:Infobox/Item')
+
+local AutoInlineIcon = Lua.import('Module:AutoInlineIcon')
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Title = Widgets.Title
+local Center = Widgets.Center
+
+---@class DeadlockItemInfobox: ItemInfobox
+local CustomItem = Class.new(Item)
+local CustomInjector = Class.new(Injector)
+
+local CATEGORY_DISPLAY = {
+	['weapon'] = 'Weapon [[Category:Weapon Items]]',
+	['vitality'] = 'Vitality [[Category:Vitality Items]]',
+	['spirit'] = 'Spirit [[Category:Spirit Items]]',
+}
+
+local DEFAULT_ATTRIBUTE_DISPLAY_FUNCTION = '_positiveConcatedArgsForBase'
+
+---@param frame Frame
+---@return Html
+function CustomItem.run(frame)
+	local item = CustomItem(frame)
+	item.args.image = item.args.image or 'Deadlock_gameasset_Item ' .. item.args.itemname .. '.png'
```

Comment:
this needed for darkmode version too?

Reply:
Currently for Deadlock the item assets we're uploading should be allmode